tests/test_helpers.py

```diff
@@ -0,0 +1,20 @@
+from unittest import mock
+import datetime
+
+import pytest
+
+class MockCtx(mock.Mock):
+    @pytest.mark.asyncio
```

Comment:
what if we put the functionality of `create_mock_ctx` in an `__init__(self, mockSend)` here

Reply:
Ok now this suggestion is what I'm talking about!